store/resources.js

```diff
@@ -1,76 +1,64 @@
-            logo: require("../assets/logos/Startup-Edmonton-Logo-300x116.png"),
-            linkTo: "https://www.startupedmonton.com/programs",
-            description:
-                "FYI Startup Edmonton has become Edmonton Unlimited and has a new space! See card the next card.",
-        },
-        edmontonUnlimited: {
-            name: "Edmonton Unlimited",
-            logo: require("../assets/logos/edmontonUnlimited.svg"),
-            linkTo: "https://edmontonunlimited.com",
-            description:
-                "Edmonton Unlimited provides free programming and resources that Edmonton's innovators, big thinkers and problem solvers need to turn their ideas into solutions.",
-        },
-        erin: {
-            name: "Mentoring from ERIN",
-            logo: require("../assets/logos/ERIN+Logo.jpeg"),
-            linkTo: "https://www.edmontonrin.ca",
-            description:
-                "Any business, whether a freshly-launched start-up or a midsize firm scaling up its offerings, depends on the wisdom and counsel of peers to reach the next level, including high-quality business coaches and mentors.",
-        },
-        startupTNT: {
-            logo: require("../assets/logos/StartUpTNTLogo.jpg"),
-            linkTo: "https://www.startuptnt.com/happy-hour",
-            description:
-                "Startup TNT believes that having fun together is the starting point for community building. Whether you are in Edmonton, Calgary or Saskatoon, grab a drink (on us) and join us at our weekly Thursday Night Tradition - Happy Hour.",
-            meetingTime: "Every Thursday 6-9pm.",
-        },
-        techAlberta: {
-            name: "Technology Alberta",
-            logo: require("../assets/logos/techAlberta.png"),
-            linkTo: "https://technologyalberta.com",
-            description:
-                "Technology Alberta exists to nurture a dedicated community of people and organizations whose coordinated and collaborative efforts help to make Alberta's entrepreneurial advanced technology companies the global vendors of choice. We achieve our goal through entrepreneur led and community driven events, programs, and initiatives.",
-        },
-    },
-});
-
-export const getters = {
-    getAllResources: (state) => {
-        return state.resources;
-    },
-    // Used to get a limited number of resources for the home page based on the card limit
-    getLimitedResources: (state) => (cardLimit) => {
-        const allResources = {};
+import { defineStore } from "pinia";
 
-        const eventKeys = Object.keys(state.resources);
-
-        for (
-            let index = 0;
-            index < cardLimit && index < eventKeys.length;
-            index++
-        ) {
-            const key = eventKeys[index];
-            allResources[key] = state.resources[key];
-        }
+export const useResourcesStore = defineStore("resources", {
+    state: () => ({
+        resources: {
```

Comment:
Similar to the events, if you make resources an array of resource objects rather than using a nested object, you can avoid the usage of Object.values().